### ISSUE TITLE
Upstream has moved

### ISSUE THREAD

**Issue by hfath** [jmkeyes/adsmpipe#4]
The ftp server "ftp://www.redbooks.ibm.com/" is gone, and the adsmpipe tarball is now under

"https://www.redbooks.ibm.com/addmat/REDP3980/adsmpipe.tar.Z"

**Comment by jmkeyes** [comment]
Thanks for the heads up! In case you need to rebuild from old sources you should be able to specify `ADSMPIPE_ARCHIVE` on the command line to change the URL where the source tarball is available:

```
$ make PLATFORM=linux ADSMPIPE_ARCHIVE="https://www.redbooks.ibm.com/addmat/REDP3980/adsmpipe.tar.Z" all
```

I've pushed a branch with a fix. Please try out the branch `fix-upstream-source` and let me know if it works for you.

**Comment by jmkeyes** [comment]
This has been fixed as of commit d8f18285.
